src/views/SystemProfile/index.tsx

```diff
@@ -605,6 +619,24 @@ const SystemProfile = ({ id, modal }: SystemProfileProps) => {
           </div>
         </SummaryBox>
 
+        {/* Only display temporary edit system profile banner if sub page does not have full edit functionality */}
+        {subPageIsEditable(subpageKey) && (
+          <GridContainer className="margin-bottom-4 margin-top-2">
+            <Alert
```

Comment:
The figma shows that this Alert is "closable." Realistically it might be weird in this situation since the "close" button for the alert would show on every sub page visit.

I think it's okay the way it is, but if UX wants to keep the Alert as closable, then you can import it from the shared folder instead as
`import Alert from 'components/shared/Alert';`
and then add the property `isClosable` to it

Reply:
My eyes totally skipped over the X, I talked to Zoe and we think it's ok to leave it non-closable